web/src/compose_ui.ts

```diff
@@ -624,6 +624,48 @@ export function handle_keyup(
     rtl.set_rtl_class_for_textarea($textarea);
 }
 
+/**
+ * True if the cursor in `$textarea` sits between an opening run of ` and its
+ * still‑missing matching closer.  Works for any run‑length (`, ```, …).
+ */
+export function cursor_inside_inline_span($textarea: JQuery<HTMLTextAreaElement>): boolean {
```

Comment:
I think you want to call it an "inline code span", not an "inline span", both here and in the other comments.